python/pyfunc-server/pyfuncserver/protocol/rest/server.py

```diff
@@ -32,11 +45,15 @@ def create_application(self):
     def start(self):
         self._http_server = tornado.httpserver.HTTPServer(
             self.create_application())
-
         logging.info("Listening on port %s", self.http_port)
         self._http_server.bind(self.http_port)
         logging.info("Will fork %d workers", self.workers)
         self._http_server.start(self.workers)
+
+        for signame in ('SIGINT', 'SIGTERM'):
```

Comment:
do we need to handle the same thing for gRPC?

Reply:
Yes, we'll need to do the same thing for grpc too.